### ISSUE TITLE
Suggestion: display tags in the tag input view

### ISSUE THREAD

**Issue by Alberdi** [babybuddy/babybuddy#471]
Disclaimer: I'm totally not a UX expert nor a web frontend developer.

At the moment there's an empty line above the tag input so the added tags end up there, and that looks a bit strange when it's empty:
<img width="781" alt="Screenshot 2022-06-03 at 20 06 13" src="https://user-images.githubusercontent.com/1556349/171921899-028314c3-3a47-4e08-8016-8068b84ed8f4.png">

Instead of simply removing it, it might be interesting to combine the input field to also display the selected tags there. Something like what [Metro has](https://metroui.org.ua/tag-input.html).

**Comment by cdubz** [comment]
I agree this would be nice. It looks a good bit more complicated than the current approach though. Metro appears to use a `input`-like container, the real `input`, a `span` for each tag, and a separate entry `input` container. Happy to consider PRs for this.

**Comment by MrApplejuice** [comment]
At my 9-5 we use bootstrap-vue and [form tags](https://bootstrap-vue.org/docs/components/form-tags/). When I was originally developing this feature, I was thinking of using them, but I was not able to find them in a way so that they were suitable to be included in "vanilla JavaScript". Since there is (was?) no real webpack or JavaScript packaging in the build-toolchain it was a quite big task to include something like this :-/ (Just some background)

**Comment by cdubz** [comment]
Yeah I am very hesitant about rebuilding the development pipeline or frontend. Gulp and a mash of committed static assets has actually served Baby Buddy quite well over the years…

However I have just started a local branch to work on moving to BS5 and a more native datetime picker. I think the changes to the datetime picker make a legitimate case for opening a “v2” branch and then maybe something like frontend modernization could really be on the table.
